Makefile

```diff
@@ -42,6 +42,9 @@ osdslet: prebuild
 osdsctl: prebuild
 	go build -o $(BUILD_DIR)/bin/osdsctl github.com/opensds/opensds/osdsctl
 
+pwdEncrypter: prebuild
```

Comment:
This module should be designed as the same with `test` and `protoc` module.

Reply:
Already updated, please review again.